mono/metadata/assembly.c

```diff
@@ -3254,6 +3254,18 @@ mono_assembly_get_image (MonoAssembly *assembly)
 	return assembly->image;
 }
 
+/**
```

Comment:
This is missing documentation on memory management.

Could the returned pointer be const?